manifest.json

```diff
@@ -39,5 +39,5 @@
       "http://api.openweathermap.org/"
    ],
    "update_url": "https://clients2.google.com/service/update2/crx",
-   "version": "3.4"
+   "version": "4.0"
```

Comment:
Incremented version number to 4.0 as discussed.

Reply:
👍